internal/terraform/lang/datasource_block.go

```diff
@@ -41,6 +41,12 @@ func (f *datasourceBlockFactory) BlockType() string {
 	return "data"
 }
 
+func (f *datasourceBlockFactory) Documentation() string {
+	return "A data block requests that Terraform read from a given data source and export the result under the " +
```

Comment:
I just kinda grabbed this off the docs site, we can iterate on them, but wanted to put something.